Challo/Challo/Persistence/Stores/Guide/GuideStore.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  GuideStore.swift
+//  Challo
+//
+//  Created by Kester Ng on 13/4/21.
+//
+
+class GuideStore {
+    private let guideModelConvertor: GuideModelConvertor
+    private let guideRepository: GuideDetailsRepositoryProtocol
+    
+    init(guideModelConvertor: GuideModelConvertor,
```

Comment:
`GuideStore` has its convertor injected via constructor, but other stores like `AreaStore` and `MapStore` constructs its own convertor. Should we be consistent and just have them injected?

Reply:
I havent changed them yet LOL, will change them later